packages/perspective-jupyterlab/test/jupyter/utils.js

```diff
@@ -109,15 +112,23 @@ module.exports = {
             visible: true,
         });
 
-        // wait for a cell to be active
-        await page.waitForSelector(
-            '.jp-Notebook-ExecutionIndicator:not([data-status="idle"])',
-            {timeout: 3000}
-        );
+        // if theres a "select kernel" dialog, just click it
+        try {
+            const button = await page.waitForSelector(
+                "div.jp-Dialog-content > div.jp-Dialog-footer > button.jp-mod-accept",
+                {timeout: 5000}
+            );
+            if (button) {
+                button.click();
+            }
+        } catch {
+            // ignore
+        }
```

Comment:
Why does this revert #1823?  Does this not happen because the notebook kernel does not match the new one added above?

Reply:
the notebook kernel specified in our original jupyter tests can fail to match the kernel on the machine, especially if they change the name of the default kernel (as they have recently).